saleor/static/js_src/dashboard.jsx

```diff
@@ -0,0 +1,78 @@
+var SORT_INFO = [ { name: 'id', dir: 'asc'}];
+
+class DataTable extends React.Component {
+    state = {
+        data: this.props.data
+    };
+
+    handleSortChange(sortInfo) {
+        SORT_INFO = sortInfo;
+
+        this.setState({data: sort(this.props.data)});
+    };
+
+    handleColumnOrderChange(index, dropIndex) {
+        var col = columns[index];
+        columns.splice(index, 1); //delete from index, 1 item
+        columns.splice(dropIndex, 0, col);
+        this.setState({})
```

Comment:
Also, what is the point of setting an empty state? There is a `forceUpdate()` method if you really need it but `columns` definitely should be part of the state. Components should never depend on data outside of their state and props (here `columns` seems to be global :skull:).